index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>DRON PAGE - NEW YEAR</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="css/output.css" />
+</head>
+
+<body>
+  <section class="hero">
+    <div class="background">
```

Comment:
there is no need for "background" element since it's css can be passed to "hero"